BParkingNano/src/classes.h

```diff
@@ -3,15 +3,14 @@
 
 #include "DataFormats/Common/interface/Wrapper.h"
 #include "TrackingTools/TransientTrack/interface/TransientTrack.h"
-#include "PhysicsTools/BParkingNano/plugins/KinVtxFitter.h"
-#include <vector>
-
+#include "PhysicsTools/BParkingNano/src/KalmanVertexFitter.h"
+#include "MagneticField/ParametrizedEngine/src/OAEParametrizedMagneticField.h"
 
 namespace {
   struct dictionary {
       std::vector<reco::TransientTrack> ttv;
       edm::Wrapper<std::vector<reco::TransientTrack> > wttv; 
-      edm::Wrapper<std::vector<KinVtxFitter> > wkv;
```

Comment:
Have you checked that one could still run the builder without this line?

Reply:
ops! I am afraid it won't work. I'll check that, but it will take a little time cause i messed up with the master locally and have to reinstall everything